x-pack/plugins/security_solution/public/management/components/endpoint_responder/command_render_components/status_action.tsx

```diff
@@ -242,7 +242,7 @@ export const EndpointStatusActionResult = memo<
                 'xpack.securitySolution.endpointResponseActions.status.lastActive',
                 { defaultMessage: 'Last active' }
               )}
-              value={endpointDetails.metadata['@timestamp']}
+              value={endpointDetails.last_checkin || endpointDetails.metadata['@timestamp']}
```

Comment:
yeah, move this to the server side when populating `last_checkin`. A few more areas in this PR do this `or`, so those should be adjusted as well.

Reply:
Oh yeah! Why didn't I think of that. Will make the change and ask for another review 😁

Reply:
So we would end up with `last_checkin: string` meaning it would always be present

Reply:
yeah absolutely 😄

Reply:
done 0575e8e5a5384a2bc59a7dd2553752a407fc5fc3